openshift_tools/monitoring/hawk_client.py

```diff
@@ -76,7 +76,11 @@ def push_metrics(self, unique_metric_list):
             _id = metric.host
             key = '{0}/{1}/{2}'.format(_type, _id, metric.key)
 
-            if isinstance(value, basestring):
+            if "heartbeat" in metric.key:
```

Comment:
@twiest because I don't exactly understand how ops-runner work, I am not sure if I need to catch all heartbeat metrics, or just the "heartbeat.ping" ones.
Current code sends all following metrics from my node to hawkular:

```
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.timeout[ozc.send.synthetic.heartbeat.quick]
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.timeout[ozc.send.heartbeat.full]
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.exitcode[ozc.send.heartbeat.full]
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.timeout[ozc.send.heartbeat.quick]
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.exitcode[ozc.send.synthetic.heartbeat.quick]
key:  node/undefcid-synthetic/heartbeat.ping
key:  node/vm-48-124.eng.lab.tlv.redhat.com/heartbeat.ping
key:  node/vm-48-124.eng.lab.tlv.redhat.com/disc.ops.runner.command.exitcode[ozc.send.heartbeat.quick]

```
Value is `up` for all.

I would think that explicit "heartbeat.ping" is enough, but it's up to you.
Note that current implementation will eliminate all Gauge metrics that contain "heartbeat" from hawkular.